src/pages/PasswordReset/index.tsx

```diff
@@ -0,0 +1,171 @@
+            role="dialog"
+            aria-modal="true"
+            aria-labelledby="modal-headline"
+          >
+            <button
+              className="absolute top-2 right-2 p-2 text-gray-700 hover:text-gray-900 focus:outline-none"
+              onClick={handleClose}
+            >
+              <img className="w-6 h-6" src={closeIcon} alt="Modal Close Icon" />
+            </button>
+
+            <div className="bg-white p-6 space-y-8 rounded-lg shadow-xl">
+              <div className="m-5">
+                <h2 className="text-2xl mb-5 font-semibold text-gray-900 text-center">
+                  Create a New Password
+                </h2>
+                <form onSubmit={handleFormSubmit}>
+                  <input
+                    type="password"
+                    name="password"
+                    placeholder="Enter your new password"
+                    value={newPassword}
+                    className="bg-gray-50 border mb-5 border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5"
+                    onChange={handleNewPasswordChange}
+                    pattern="^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d@$!%*#?&]{8,20}$"
+                    title="Please include at least 1 uppercase character, 1 lowercase character, and 1 number."
+                    required
+                  />
+                  <input
+                    type="password"
+                    name="confirmPassword"
+                    placeholder="Confirm your new password"
+                    value={confirmPassword}
+                    className="bg-gray-50 border mb-5 border-gray-300 text-gray-900 text-sm rounded-lg focus:ring-blue-500 focus:border-blue-500 block w-full p-2.5"
+                    onChange={handleConfirmPasswordChange}
+                    pattern="^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d@$!%*#?&]{8,20}$"
+                    title="Please include at least 1 uppercase character, 1 lowercase character, and 1 number."
+                    required
+                  />
+                  <p className="text-gray-600 mb-5">
+                    Your password must be 8-20 characters long, contain letters
+                    and numbers, and must not contain spaces, special
+                    characters, or emoji.
+                  </p>
+                  <div>
+                    {loading ? (
+                      <div className="flex justify-center items-center px-4 py-2">
+                        <>
+                          <Loading />
+                        </>
+                      </div>
+                    ) : (
+                      <button
+                        type="submit"
+                        className="w-full px-4 py-2 text-base font-semibold text-center text-white bg-blue-500 rounded-lg hover:bg-blue-700 focus:outline-none focus:ring-2 focus:ring-blue-400 focus:ring-opacity-50 cursor-pointer"
```

Comment:
Check mobile responsiveness as well

Reply:
![image](https://github.com/sef-global/scholarx-frontend/assets/48531182/79647613-2721-4b6c-98cc-f90526a2bc3e)
is it good ?